Detections/OracleDatabaseAudit/OracleDBAuditSelectOnManyTables.yaml

```diff
@@ -0,0 +1,33 @@
+id: 75024e1c-26e7-4e73-821d-95e5decdd8db
+name: OracleDBAudit - Unusual user activity on multiple tables 
+description: |
+  'Detects when user queries many tables in short period of time.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: OracleDatabaseAudit
+    dataTypes:
+      - Syslog
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Collection
+query: |
+  let lbtime = 3m;
+  let tbl_threshold = 10;

```

Comment:
If you need this to be 10 in 5m, then in the summarize use bin(TimeGenerated, 5m) and this will give you the count for every 5 mins of the hour lookback.  Then you can match when only greater than you threshold of 10 here.

Reply:
updated